### PR TITLE
Text style changes

### LINKED ISSUE #962: the text style of code doesn't follow the Design Library definition
## Describe the bug
The text style of code in the attribution tool, html tab, has a different text style.

## Expected behavior
The text style should be `code-normal` as it indicates [the Design Library](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=0%3A110&viewport=-651%2C-1185%2C0.6580211520195007).

## Screenshots
Screen Shot 2020-05-14 at 14 02 54

### PR DESCRIPTION
**Pull request by akanksha-v**
Closes #962 

I have changed the text style code to code-normal as specified in the design-libray.
WhatsApp Image 2020-05-17 at 1 49 41 PM

### CODE REVIEW

File: src/components/ImageDetails/CopyLicense.vue [hunk 1]
```diff
@@ -55,7 +55,7 @@
         </div>
         <div :class="tabClass(1, 'tabs-panel')">
           <textarea id="attribution-html"
-                    class="textarea is-family-monospace is-paddingless"
+                    class="textarea is-family-JetBrains-Mono is-paddingless has-text-weight-normal"
```
Comment (brenoferreira) [inline src/components/ImageDetails/CopyLicense.vue:58]: You should use the `.monospace` class, see the vocabulary docs https://cc-vocabulary.netlify.app/?path=/docs/tokens-typography--figma. Also no need to add this `. has-text-weight-normal` class

Comment (akanksha-v) [reply src/components/ImageDetails/CopyLicense.vue:58]: @brenoferreira so what exactly needs to be changed, I applied the changes that match the text with [code-normal](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=445%3A1053) as given in #962

Comment (brenoferreira) [reply src/components/ImageDetails/CopyLicense.vue:58]: The issue says the code needs to look like the code font in the design library, which uses a monospaced font. This `is-family-JetBrains-Mono` class doesn't exist in Vocabulary. The `.has-text-weight-normal` isn't necessary either. Hope that clears it up for you

**Comment by zackkrida** [comment]
Hello @akanksha-v, are you interested in finishing this work? If so, the `textarea` should only need the following: `class="textarea is-family-monospace is-paddingless"`. Thanks for working on this!

**Comment by zackkrida** [comment]
Hi @akanksha-v, I lied to you! Oops. I think I figured out why we were _both_ confused. `.is-family-monospace` gives us the larger monospace font, and `.monospace` gives us the desired "code normal" font size. Could you switch it and I'll merge this today? Thanks

**Pull request merged**